### PR TITLE
ALL: Unify messages wording

### PR DESCRIPTION
**Pull request by BenCastricum**
While redoing the dutch translation I came up with these editorial suggestions. Feedback appreciated.

### CODE REVIEW

File: engines/advancedDetector.cpp [hunk 1]
```diff
@@ -334,7 +334,7 @@ void AdvancedMetaEngine::reportUnknown(const Common::FSNode &path, const ADFileP
 	Common::String report = Common::String::format(
 			_("The game in '%s' seems to be unknown.\n"
 			  "Please, report the following data to the ScummVM team along with name\n"
```
Comment (csnover) [inline engines/advancedDetector.cpp]: “along with *the* name”

File: engines/advancedDetector.cpp [hunk 2]
```diff
@@ -334,7 +334,7 @@ void AdvancedMetaEngine::reportUnknown(const Common::FSNode &path, const ADFileP
...
-			  "of the game you tried to add and its version/language/etc.:"), path.getPath().c_str()) + "\n";
```
Comment (csnover) [inline engines/advancedDetector.cpp]: “…add *and* its version, …”

File: engines/scumm/detection.cpp [hunk 3]
```diff
@@ -1120,7 +1120,7 @@ Common::Error ScummMetaEngine::createInstance(OSystem *syst, Engine **engine) co
 		md5Warning = _("Your game version appears to be unknown. If this is *NOT* a fan-modified\n"
 		               "version (in particular, not a fan-made translation), please, report the\n"
 		               "following data to the ScummVM team along with name of the game you tried\n"
-		               "to add and its version/language/etc.:\n");
```
Comment (csnover) [inline engines/scumm/detection.cpp]: “add” should still be in the sentence. Best written the same way as suggested for the same content in `advancedDetector.cpp`: “…to add *and* its version, …”

File: engines/sherlock/detection.cpp [hunk 4]
```diff
@@ -68,8 +68,8 @@ static const ADExtraGuiOptionsMap optionsList[] = {
 	{
 		GAMEOPTION_ORIGINAL_SAVES,
 		{
-			_s("Use original savegame dialog"),
-			_s("Files button in-game shows original savegame dialog rather than the ScummVM menu"),
+			_s("Use original save/load screens"),
+			_s("Use the original save/load screens, instead of the ScummVM ones"),
```
Comment (csnover) [inline engines/sherlock/detection.cpp]: The comma may be left out here.

**Comment by BenCastricum** [comment]
Thanks @csnover , I changed to prefix to "ALL"  as sev already did for the PR, and have integrated your suggestions as well.

**Comment by criezy** [comment]
Those changes look good to me.

**Comment by sev-** [comment]
Great! Merging.

**Pull request merged**